theme.toml

```
@@ -5,7 +5,7 @@ description = "A Hugo theme for technical documentation sites"
 homepage = "https://docsy.dev"
 tags = ["documentation", "multilingual", "customizable", "responsive", "docs"]
 features = []
-min_version = 0.110.0
+min_version = 0.110
```

Might the fix instead be to make it a string?
```suggestion
min_version = "0.110.0"
```